docs/pt-br/strict.md

```diff
@@ -0,0 +1,26 @@
+# Strict Mode (ou Modo Estrito)
+
+Para habilitar o strict mode, simplesmente passe `strict: true` quando for criar a store Vuex:
+
+``` js
+const store = new Vuex.Store({
+  // ...
+  strict: true
+})
+```
+
+No strict mode, toda vez que um estado Vuex é mutado fora de um handler de mutação, um erro vai ser disparado. Isso assegura que todas as mutações de estado sejam explicitamente rastreadas por ferramentas de debug.
+
+### Desenvolvimento Vs. Produção
+
+**Não habilite strict mode quando for dar deploy em produção!** Strict mode roda um watcher síncrono profundo na árvore de estados para detectar mutações inapropriadas, e pode consumir bastante recursos quando você fizer uma quantidade grande de mutações no estado. Assegure-se de desligá-lo em produção para evitar esse gasto em performance.
```

Comment:
Suggestion:

**Não habilite o modo estrito quando for publicar em produção!** Modo estrito executa um observador síncrono profundo na árvore de estados para detectar mutações inapropriadas, e pode consumir muitos recursos se você fizer uma quantidade grande de mutações no estado. Assegure-se de desligá-lo em produção para evitar esta perda de desempenho.